ui/shared/ToastMessage.qml

```diff
@@ -32,7 +32,7 @@ Popup {
         // Reset props
         iconColor = Style.current.primary
         iconRotates = false
-        linkText = defaultLinkText
+        root.linkText = defaultLinkText
```

Comment:
Using just `linkText` results in an error because `linkText` here is not only the name of a property of this component, it also happens to be the ID of some other component.